CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -90,7 +90,7 @@ typealias LumaListener = (luma: Double) -> Unit
 class CameraFragment : Fragment() {
 
     private lateinit var container: ConstraintLayout
-    private lateinit var viewFinder: PreviewView
+    private lateinit var previewView: PreviewView
```

Comment:
Please don't change the name of this variable, saying it's a viewfinder is more descriptive than just repeating the class in the variable name

Reply:
I'm changing that.